app/src/main/java/com/willowtree/vocable/settings/AddPhraseViewModel.kt

```diff
@@ -0,0 +1,103 @@
+package com.willowtree.vocable.settings
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.willowtree.vocable.BaseViewModel
+import com.willowtree.vocable.presets.PresetCategories
+import com.willowtree.vocable.presets.PresetsRepository
+import com.willowtree.vocable.room.CategoryPhraseCrossRef
+import com.willowtree.vocable.room.Phrase
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.koin.core.inject
+import java.util.*
+
+class AddPhraseViewModel : BaseViewModel() {
+
+    companion object {
+        private const val PHRASE_UPDATED_DELAY = 2000L
+    }
+
+    private val presetsRepository: PresetsRepository by inject()
+
+    private val liveMySayingsList = MutableLiveData<List<Phrase>>()
+    val mySayingsList: LiveData<List<Phrase>> = liveMySayingsList
```

Comment:
What are these values going to be used for?

Reply:
These were holdovers from a copy/paste, they need to be removed I think